netsuitesdk/api/vendor_subsidiary_relationships.py

```diff
@@ -0,0 +1,14 @@
+from .base import ApiBase
+
+
+class VendorSubsidiaryRelationship(ApiBase):
+    SIMPLE_FIELDS = [
+    ]
+
+    RECORD_REF_FIELDS = [
+        'entity',
+        'subsidiary',
+    ]
```

Comment:
I actually don't know what the point of these are, seems to include a bunch of fields I don't specify anyway.

Reply:
this is primarily used for building request payloads